modules/_labs/color-picker/react/lib/ColorPicker.tsx

```diff
@@ -191,4 +227,5 @@ const ColorPicker = ({
   );
 };
 
+ColorPicker.defaultColorSet = defaultColorSet;
```

Comment:
Exporting the `defaultColorSet` in a namespaced manner here. Thanks @NicholasBoll